Source/ACE.Server/WorldObjects/Gem.cs

```diff
@@ -77,10 +73,10 @@ public override void UseItem(Player player)
                     castMessage = "The gem casts Asheron's Lesser Benediction on you";
                 */
 
-                player.CreateItemSpell(Guid, (uint)SpellDID);
+                player.CreateItemSpell(this, (uint)SpellDID);
 
                 // always consume gem?
-                player.TryRemoveFromInventoryWithNetworking(this);
+                player.TryConsumeFromInventoryWithNetworking(this);
 
                 player.SendUseDoneEvent();
                 return;
```

Comment:
Should have included a little more in the code reference.  For gems with unlimited usage, they should be excluded from player.TryConsumeFromInventoryWithNetworking(this).

Reply:
gotcha!, should be fixed now *i think*